src/components/Game/Game.js

```diff
@@ -65,28 +65,24 @@ class GameView extends Component {
             return;
         }
 
-        let newState;
+        const newState = {
+            dragDisabled: dragDisabledOnResizeEnd,
+        };
 
-        if (isMobile()) {
-            newState = {
-                currentAnswer: [...letters],
-                isSubmitVisible: true,
-                letters: this.setAllLettersActiveState(letters, false),
-                dragDisabled: this.getDragDisabled(),
-            };
+        if (dragDisabledOnResizeEnd) {
+            newState.letters = this.setAllLettersActiveState(letters, false);
+            newState.currentAnswer = [...letters];
+            newState.isSubmitVisible = true;
```

Comment:
you can also do
```
newState = {
    ...newState,
    letters: …,
    currentAnswer: …,
    isSubmitVisible: …,
};
```
nicer syntax?

obviously it's not a blocker, just code styling suggestion